guides/common/modules/proc_enabling-tuned-profiles.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Enabling_Tuned_Profiles_{context}"]
+= Enabling Tuned Profiles
+
+{RHEL} 7 enables the tuned daemon by default during installation.
+On bare-metal, {Team} recommends to run the `throughput-performance` tuned profile on {ProjectServer} and {SmartProxies}.
+On virtual maschines, {Team} recommdsn to run the `virtual-guest` profile.
+
+.Procedure
+. Check if `tuned` is running:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl status tuned
+----
+. If `tuned` is not running, enable it:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl enable tuned
```

Comment:
`enable` does not start the service. It just enables it (which means: start it at boot). If you want to start it
```suggestion
. If `tuned` is not running, enable it:
+
[options="nowrap" subs="+quotes,attributes"]
----
# systemctl enable --now tuned
```

Reply:
Applied.